src/tree-view/treeViewComponent.ts

```diff
@@ -0,0 +1,81 @@
+const $ = (<any> window).$;
+const Promise = (<any> window).Promise;
+
+export class TreeViewController {
+  public tree;
+  public reselect;
+  public data;
+  public name : string;
+  public onSelect;
+  public lazyLoad;
+
+  /*@ngInject*/
+  constructor(private $element, private $timeout, private $http) {}
```

Comment:
`$timeout` and `$http` is not used in this component aren't they? So we can remove them.